### DISCUSSION TITLE
Transfer to another server

### DISCUSSION THREAD

**Discussion by n1ght-hunter** [firefly-iii/firefly-iii#7993] [Firefly III]
whats the easest way to transfer a firfly iii docker instance from one sever to another

**Comment by JC5** [comment]
copy paste

**Comment by JanFeld** [comment]
Actually i'd like to transfer my build (docker-compose) too. 
I'm not sure how well it handles the database transfer.

Can a imported database using a dump (mysqldump) of my current DB be used by a new instance on another server, if the setup is the same (docker-compose, .env, etc.)?

**Reply by JC5** [reply]
You can either copy/paste the named volumes, or use mysqldump. 

1. Names volumes must have the correct access rights also copied over.
2. Keep in mind mariadb no longer ships with mysqldump
